src/components/WalletNotFound.vue

```diff
@@ -10,12 +10,12 @@
         </p>
       </div>
       <div class="link-container">
-        <a
-          class="link"
-          href="https://github.com/witnet/witnet-rust/tree/master/wallet"
-          target="_blank"
-        >
-          https://github.com/witnet/witnet-rust/tree/master/wallet
+        <a class="link" href="https://t.me/witnetio" target="_blank">
+          Witnet Community Telegram Channel
```

Comment:
I would add the links within a single paragraph, something like "Contact for support in our [Telegram](url) or [Discord](url) community channel".

<img width="652" alt="Screenshot 2024-04-01 at 09 59 37" src="https://github.com/witnet/sheikah/assets/16032491/2c3c10e8-9e1e-4513-8e57-d8fb2527a5ea">